### PR TITLE
Implement favicon

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Screen Shot 2020-03-26 at 4 01 41 PM
Screen Shot 2020-03-26 at 4 01 33 PM

### CODE REVIEW

File: packages/apps/webpack.config.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
       new HtmlWebpackPlugin({
         inject: true,
         template: path.join(context, `${hasPublic ? 'public/' : ''}${name}.html`),
-        PAGE_TITLE: 'Polkadot/Substrate Portal'
+        PAGE_TITLE: 'CENNZnet/Substrate Portal'
```
Comment (holygits) [inline packages/apps/webpack.config.js]: CENNZnet Portal

Comment (KarishmaBothara) [reply packages/apps/webpack.config.js]: sure

File: packages/react-components/src/ChainImg.tsx [hunk 2]
```diff
@@ -83,6 +84,6 @@ function ChainImg ({ className, logo, onClick }: Props): React.ReactElement<Prop
 }
 
 export default styled(ChainImg)`
```
Comment (holygits) [inline packages/react-components/src/ChainImg.tsx]: Should this icon/logo switch dynamically rather than replace 💭

Comment (KarishmaBothara) [reply packages/react-components/src/ChainImg.tsx]: According what Ben mentioned in the comment..
The hamburger button is for mobile view when you keep narrowing down from the above view, and you would see a small black logo on the left top corner. And what we need to do is replacing that dark image to be hamburger one. We may need to make some changes around here - https://github.com/cennznet/apps/blob/develop/packages/react-components/src/ChainImg.tsx#L14

**Pull request merged**